conf/messages.cy

```diff
@@ -575,13 +568,14 @@ uploadSupportingDocuments.checkYourAnswersLabel = Dogfennau ategol wedi’u huwc
 uploadSupportingDocuments.change.hidden = upload a document
 
 uploadSupportingDocuments.error.entitytoolarge = The selected file must be smaller than {0}MB
-uploadSupportingDocuments.error.entitytoosmall = The selected file is empty
-uploadSupportingDocuments.error.invalidargument = Select a file to upload
-uploadSupportingDocuments.error.duplicate = You can only upload distinct files
-uploadSupportingDocuments.error.failed = The selected file failed to upload
-uploadSupportingDocuments.error.quarantine = The selected file contains a virus
-uploadSupportingDocuments.error.rejected = Mae angen i’r ffeil dan sylw fod ar ffurf pdf neu jpeg
-uploadSupportingDocuments.error.unknown = There was a problem uploading your file
+fileUpload.error.entitytoolarge = The selected file must be smaller than {0}MB
+fileUpload.error.entitytoosmall = The selected file is empty
+fileUpload.error.invalidargument = Select a file to upload
+fileUpload.error.duplicate = You can only upload distinct files
+fileUpload.error.failed = The selected file failed to upload
+fileUpload.error.quarantine = The selected file contains a virus
+fileUpload.error.rejected = Mae angen i’r ffeil dan sylw fod ar ffurf pdf neu jpeg
```

Comment:
We recently did this story: 
https://jira.tools.tax.service.gov.uk/browse/ARSSTB-251
where the casing of pdf/jpg was changed to PDF/JPG. Would something similar be required for Welsh messages?

Reply:
from discussion - this will be dealt with in a separate PR as there are more occurrences throughout messages files